packages/usa-input-mask/src/styles/_usa-input-mask.scss

```
@@ -0,0 +1,28 @@
+@use "uswds-core" as *;
+
+.usa-input-mask {
+  position: relative;
+  line-height: 1;
+
+  &--content {
+    position: absolute;
+    left: 2px;
+    top: 0;
+    pointer-events: none;
+    color: color("gray-50");
+    font-family: family("sans");
+    border: 1px solid transparent;
+    i {
+      visibility: hidden;
+    }
+  }
+}
+
+usa-masked,
+.usa-input-mask--content {
+  padding: units(1);
+  line-height: 1.4;
+  font-size: size('body', 'sm');
```

`.usa-input` typography is defined with the `block-input-general` placeholder, which uses form settings. Can we replace these typography declarations with the same placeholder?

```suggestion
@extend %block-input-general;
```

---

Done